### ISSUE TITLE
NVDA does not read google talks windows as re

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4981]
**Reported by sumandogra on 2015-03-18 10:25**
Google Talks

Usually in Google talks the contact list is not read out with the PC cursor
In the chat window the text is not read with the PC cursor
In both the scenarios screen review is very much unpredictable.
Some times on 32 bit and XP machines a user can read the text with the screen reviewer however in some other machines and mostly on the 64 bit machines it reads only few words sometimes only the top 2 lines of the window.

**Comment by dkager** [comment]
Closing because Google Talk has been discontinued (and because this issue never got a response).
